### PR TITLE
add a check for HF block 1920000

### PR DESCRIPTION
**Pull request by splix**

### CODE REVIEW

File: ethereumj-core/src/main/java/org/ethereum/net/eth/handler/Eth62.java [hunk 1]
```diff
@@ -676,6 +679,17 @@ private boolean isValid(BlockHeadersMessage response, GetBlockHeadersMessageWrap
             }
         }
 
+        // check that these headers belong to classic chain
+        for (BlockHeader header: headers) {
+            if (!hardForkRule.validate(header)) {
```
Comment (bart-w0) [inline ethereumj-core/src/main/java/org/ethereum/net/eth/handler/Eth62.java]: Isn't hardForkRule null here? Do we need to initialise it first (or autowire)?

Comment (splix) [inline ethereumj-core/src/main/java/org/ethereum/net/eth/handler/Eth62.java]: You're right, forgot to initialize it

Comment (splix) [inline ethereumj-core/src/main/java/org/ethereum/net/eth/handler/Eth62.java]: (repeating) _I'll never do "quick cleanup" before commit_...

**Comment by bart-w0** [comment]
Looks good. :+1:

My main concern against such HF code was that each client would have to perform some checks on each block. At least you guard expensive hash comparison with header number, so that's good.

File: ethereumj-core/src/main/java/org/ethereum/net/eth/handler/Eth62.java (hunk 1 above)
Comment (splix) [inline ethereumj-core/src/main/java/org/ethereum/net/eth/handler/Eth62.java]: fixed in [d0fdcfb]

**Comment by bart-w0** [comment]
:+1: @splix see Trello board before we merge it in.

**Pull request merged**